src/main/java/controller/LadderGame.java

```diff
@@ -4,20 +4,19 @@
 import view.InputView;
 import view.OutputView;
 
-public class GameController {
+public class LadderGame {
 
-    public void ladderGameStart() {
-        Ladder ladder = gameReady();
+    public void start() {
+        Ladder ladder = init();
         OutputView.printLadder(ladder);
     }
 
-    private Ladder gameReady() {
+    private Ladder init() {
+        InputView.createScanner();
```

Comment:
```suggestion
        InputView.init();
```
inputView 스캐너를 열고 닫는 걸 바깥에서 알 필요는 없을 것 같습니다.